<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QgsProjectionSelectorBase</class>
 <widget class="QWidget" name="QgsProjectionSelectorBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Projection Selector</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>1</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Projection</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="lstCoordinateSystems" >
        <column>
         <property name="text" >
          <string>Coordinate System</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Spatial Reference Id</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="teProjection" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>1</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFormatting" >
         <set>QTextEdit::AutoBulletList</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="title" >
         <string>Search</string>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="1" column="5" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="3" colspan="2" >
          <widget class="QPushButton" name="pbnFind" >
           <property name="text" >
            <string>Find</string>
           </property>
           <property name="default" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3" >
          <widget class="QLineEdit" name="leSearch" />
         </item>
         <item row="0" column="0" >
          <widget class="QRadioButton" name="radSRID" >
           <property name="text" >
            <string>Postgis SRID</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QRadioButton" name="radEPSGID" >
           <property name="text" >
            <string>EPSG ID</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2" >
          <widget class="QRadioButton" name="radSRSID" >
           <property name="text" >
            <string>QGIS SRSID</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4" >
          <widget class="QRadioButton" name="radName" >
           <property name="text" >
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
